storage/migrations/14_block_meta.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+ALTER TABLE chain.nodes
+    ADD COLUMN consensus_pubkey_address TEXT;
+
+CREATE INDEX ix_nodes_consensus_pubkey_address ON chain.nodes (consensus_pubkey_address);
+
+ALTER TABLE chain.blocks
+    ALTER COLUMN block_hash DROP NOT NULL,
+    ALTER COLUMN time DROP NOT NULL,
+    ALTER COLUMN num_txs DROP NOT NULL,
+    ALTER COLUMN namespace DROP NOT NULL,
+    ALTER COLUMN version DROP NOT NULL,
+    ALTER COLUMN type DROP NOT NULL,
+    ALTER COLUMN root_hash DROP NOT NULL,
+    ADD COLUMN proposer_node_consensus_pubkey_address TEXT,
+    ADD COLUMN signer_node_consensus_pubkey_addresses TEXT[];
```

Comment:
Can we add a comment about why this is not `oasis_addr[]`, and what the actual values are here?

Alternatively, let's store the oasis addr here (as suggested in https://github.com/oasisprotocol/nexus/pull/646/files#r1591463407 by Andy). @jberci said (and Warren probably knows well) that we can derive this on the fly, rather than looking it up from the DB. The downside is that we're less robust if very hypothetically oasis-core were to change the derivation scheme; negligible risk, IMO.